dockerExtension.ts

```diff
@@ -6,58 +6,115 @@
 let loadStartTime = Date.now();
 
 import * as assert from 'assert';
-import * as opn from 'opn';
 import * as path from 'path';
 import * as request from 'request-promise-native';
 import * as vscode from 'vscode';
-import { AzureUserInput, callWithTelemetryAndErrorHandling, createTelemetryReporter, IActionContext, parseError, registerCommand as uiRegisterCommand, registerUIExtensionVariables, TelemetryProperties, UserCancelledError } from 'vscode-azureextensionui';
-import { ConfigurationParams, DidChangeConfigurationNotification, DocumentSelector, LanguageClient, LanguageClientOptions, Middleware, ServerOptions, TransportKind } from 'vscode-languageclient/lib/main';
+import {
+  AzureUserInput,
+  callWithTelemetryAndErrorHandling,

```

Comment:
What are you using that's causing all these diffs?  This is really non-standard formatting.

Reply:
Also, I just entered a bug that turned out to be a bug in your branch, likely a bad merge caused by all these diffs: https://github.com/Microsoft/vscode-docker/issues/600

azurecli register should look like this:
registerCommand('vscode-docker.container.start.azurecli', async function (this: IActionContext): Promise<void> { await startAzureCLI(this); });

Reply:
That's the formatting change I asked you about a couple of weeks ago. This was done by the interns. and for the most part of it, there is no change. It's just formatting.

About the bug, I never have used that command. I'm not even sure what it does, but I'll look into it.